R/estimate_grouplevel.R

```diff
@@ -92,7 +91,7 @@ estimate_grouplevel <- function(model, type = "random", ...) {
   random <- datawizard::data_relocate(random, c("Component", "Group", "Level", "Parameter"), verbose = FALSE)
 
   # Clean-up brms output
-  if (inherits(model, "brmsfit")) {
+  if (inherits(model, "brmsfit") && FALSE) {
```

Comment:
@DominiqueMakowski I'm not sure if we need this cleaning anymore? Not sure for which models you added this, but probably this is resolved in _parameters_?